size_sort.py

```diff
@@ -35,14 +36,12 @@ def main():
     # dirs= [dir for dir in dirs if isdir(dir)]
     df= pd.read_csv(infoFile)
 
-    df_parent= pd.DataFrame(columns= ['Directory', 'SizeG'])
+    df_parent= pd.DataFrame(columns= ['Directory', 'SizeG', 'Last Modified'])
     for j,dir in enumerate(dirs):
-        space= 0.
-        for i,name in enumerate(df.values[:,3]):
-            if dir in name:
-                space+=df.values[:,2][i]
+        for i,name in enumerate(df[' Directory']):
+            if dir==name+'/':
+                df_parent.loc[j]= [dir,df[' SizeG'][i],df[' Last Modified'][i]]
 
-        df_parent.loc[j]= [dir,space/2.]
 
```

Comment:
This logic wasn't right before due to the presence of a `/` at the end of names in `infoFile`. Now space is obtained directly from `infoFile`. Thus it is more reliable now.